README.rst

```diff
@@ -60,6 +60,14 @@ can use the :code:`@pytest.mark.repeat(count)` decorator:
    def test_repeat_decorator():
        pass
 
+If you want to override default tests executions order, you can use :code:`--repeat-scope`
+command line option with one of the next values: :code:`session`,  :code:`module`, :code:`class` of :code:`function`(default).
```

Comment:
Typo: 'of' instead of 'or'

Reply:
Also, `function` is not rendoring correctly, see https://github.com/gdyuldin/pytest-repeat/blob/31c85c7f9398b729c8c4abf46f491ea6512054ea/README.rst. I suspect you need a space before "(default)".